src/Controls/src/Core/PublicAPI/net-maccatalyst/PublicAPI.Shipped.txt

```diff
@@ -97,7 +97,6 @@
 ~Microsoft.Maui.Controls.AbsoluteLayout.SetLayoutFlags(Microsoft.Maui.IView view, Microsoft.Maui.Layouts.AbsoluteLayoutFlags flags) -> void
 ~Microsoft.Maui.Controls.Accelerator.Keys.get -> System.Collections.Generic.IEnumerable<string>
 ~Microsoft.Maui.Controls.Accelerator.Keys.set -> void
-~Microsoft.Maui.Controls.Accelerator.Modifiers.get -> System.Collections.Generic.IEnumerable<string>
```

Comment:
We should not modify Shipped files.

Rather copy the line into Unshipped and add a `*REMOVED*`:

```
*REMOVED*~Microsoft.Maui.Controls.Accelerator.Modifiers.get -> System.Collections.Generic.IEnumerable<string>
```